tox.ini

```
@@ -1,24 +1,27 @@
 [tox]
 envlist =
-    py{27,35,36,37,38}-django{111,20,21,22,master},
-    py{36,37,38}-django{30,31},
+    py{35,36,37,38,39}-django22,
```

I don't _think_ we need Python 3.5 here.
```suggestion
    py{36,37,38,39}-django22,
```